doc/developer/interface/custom-login/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+---
+
+
+Custom Login Page
+==========================================================
+
+The default landing / login page for IRIDA can be replaced by an institution without modifying the existing one.
+
+Create a html file called `login.html` on your files system, and save it to the path `/etc/irida/templates/`;
+
+```properties
+# Overwritten UI templates
+ui.templates=/etc/irida/templates/
+```
+
+### Use IRIDA base login form.
+
+IRIDA uses a React component as its login form and can be added to any template by adding:
+
+ * CSS (`/dist/css/login.bundle.css`) and JS (`/dist/js/login.bundle.js`) files.
+ * A `head` > `script` tag with a global `PAGE` variable containing: `BASE_URL: /*[[@{"/"}]]*/ "/"` as an attribute.
+ * The div with id `login-root` should be placed on the dom element that you want React to render the login form.
+ 
+#### Adding custom script
```

Comment:
Can we change this to `Adding custom resources`?

Also add some about adding images in here as well.  It'll likely be one of the biggest uses of this feature.

Reply:
Updated in 7a8e711a72be7eaa4dda707e1a604f9e0dbbb6fa